fetch/api/META.yml

```diff
@@ -0,0 +1,5 @@
+links:
+    - product: chrome
+      url: https://crbug.com/1033852
```

Comment:
This is probably unrelated, but leaving for now

Reply:
https://bugs.chromium.org/p/chromium/issues/detail?id=836697 would be the one to link here. I'll do just that. Linking this one too seems fine, but not super useful for the purposes of "what should we fix to improve interop?"

Reply:
Done.